pkgs/applications/misc/remnote/default.nix

```
@@ -0,0 +1,17 @@
+{ lib, fetchurl, appimageTools }:
+
+appimageTools.wrapType2 rec {
+  name = "remnote";
+  version = "1.7.6";
+
+  src = fetchurl {
+    url = "https://download.remnote.io/RemNote-${version}.AppImage";
+    sha256 = "sha256-yRUpLev/Fr3mOamkFgevArv2UoXgV4e6zlyv7FaQ4RM=";
+  };
+
+  meta = with lib; {
+    description = "A note-taking application focused on learning and productivity";
+    homepage = "https://remnote.com/";
+    license = licenses.unfree;
```

```suggestion
    license = licenses.unfree;
    platforms = platforms.linux;
```
Appimage only runs on linux. Also missing maintainer

---

Interesting; I had assumed `appimageTools` set the platforms. Anyway, both issues are fixed now.